packages/dd-trace/src/config.js

```diff
@@ -508,6 +508,17 @@ ken|consumer_?(?:id|key|secret)|sign(?:ed|ature)?|auth(?:entication|orization)?)
       true
     )
 
+    const DD_EXPERIMENTAL_API_SECURITY_ENABLED = coalesce(
+      process.env.DD_EXPERIMENTAL_API_SECURITY_ENABLED && isTrue(process.env.DD_EXPERIMENTAL_API_SECURITY_ENABLED),
+      false
+    )
+
+    const DD_API_SECURITY_REQUEST_SAMPLE_RATE = coalesce(
+      options.appsec?.apiSecurity?.requestSampling,
```

Comment:
If we add an option, we should update `index.d.ts` file and also add some test in `test.ts` file.